tests/CMakeLists.txt

```diff
@@ -75,7 +75,8 @@ set(SRC gtmain.cpp
   function/gtEnumerate.cpp
   function/gtHostif.cpp
   function/gtMMIO.cpp
-  function/gtEvent.cpp)
+  function/gtEvent.cpp
+  function/gtVersion.cpp)
```

Comment:
I see you added tests but they aren't run as part of the tests.

Reply:
Right - how do I add them there?

Reply:
So only a few tests are currently being run using a filter:
https://github.com/OPAE/opae-sdk/blob/7d4e3fb03cd78a1e8dc72329551670ab4c974c94/tests/CMakeLists.txt#L108-L118

The test is named: `test_open_common_mock_drv`

I suggest using narrower filters for the tests. For example, in the C++ tests, I added a macro to add tests with filters. Calling this macro looks like this:
https://github.com/OPAE/opae-sdk/blob/ee8a9e184a2053b6660fccf1ff3db04579c338c3/tests/CMakeLists.txt#L171-L177

And in travis it looks like:
![image](https://user-images.githubusercontent.com/22808196/36320853-8ccb6940-12fc-11e8-88cb-c7968d420cd6.png)